### ISSUE TITLE
Trailing comma in unset parse fails

### ISSUE THREAD

**Issue by archfz** [pdepend/pdepend#560]
- PHP Depend version: 2.x-dev
- PHP Version: 8.0.3
- Installation type: composer
- Operating System / Distribution & Version: (Windows 10 / CentOS 7.6 / ...)

### Current Behavior
The following fails to parse.
```php
        unset(
            $model['id'],
            $model['imageSrc'],
            $model['interiorImageURL'],
            $model['interiorImage360URL'],
            $model['vehicleVariants'],
        );
```

Unexpected token: ), line: 321, col: 9, file: /app/src/Core/Configurator/Domain/Service/ProductApi/ProductApiImportSource.php.

### Expected Behavior
Do not fail.

### Steps To Reproduce:
Above

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/pdepend/pdepend/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/pdepend/pdepend/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "Default value is not available in AST when it's an array".

**Comment by kylekatarnls** [comment]
Hello, thanks for the report. As your PHP Depend version is a branch, please precise the commit hash (source.reference in composer.lock).

**Comment by archfz** [comment]
Obviously I tried the latest commit :smile:

**Comment by kylekatarnls** [comment]
That's not obvious, you may have a composer.lock file that keep you on an older "latest" version of the branch. Also we might push new commits after you create the issue, so we have to check the date of your post and guess what was the "latest" at that point.

A branch is not a version, it's a line, tags and hashes are unambiguous frozen versions of the code, that get all doubts out of the way.

I will assume `eb459a2ca70a1e752e92a6c59c140235535a8cbb` and will try to reproduce.

**Comment by archfz** [comment]
Look. It could be worse. I could have written `2.10.0` which is the latest release. And in that case you'd know even less.

**Comment by kylekatarnls** [comment]
Not worse actually. If you actually use 2.10.0 and give us this number, it's perfect, we download this exact version, try the same code and see if we reproduce.

If yes => we can work on this version, if we get it fixed locally, the same patch should fix it for you and everyone.
If no => the error depends on the context (OS, dependencies, an other part of the code that was scanned etc.)

If you give a range of commits and we don't reproduce with 1 commit of the range, maybe it's because it's a regression after this commit or was yet fixed. So this is one more thing to check that is not needed if you give a version, even if it's 2.7.0, or an old hash, what ever the version it is, what's matter is to be sure we run the same code as you when we debug PHP Depend.

**Comment by kylekatarnls** [comment]
I reproduce it and even with the simplest `unset($i,)` form. It sounds related to `unset()` only, replacing it with any other function works fine.

I also checked consistency with other special keywords, and trailing comma sounds not valid in PHP syntax for all other special functions (`eval('echo "FOO";',)` throws a parse error) or for keywors where the parentheses are optional like in `echo($i,)` / `print($i,)` nor in statements like `if ($i,)`)

So it sounds like here we just have a special case to handle for `unset()` which would be the only keyword to really behave like a function regarding trailing commas.

**Comment by kylekatarnls** [comment]
You can test my fix for that using `composer require pdepend/pdepend:dev-fix/trailing-comma --dev`

**Comment by archfz** [comment]
Cannot install for some reason `Could not find a matching version of package dev-fix/trailing-comma`. Installing dev-master works on the other hand. Composer version 2.0.12

**Comment by kylekatarnls** [comment]
It exists, maybe try to clear your cache with `composer clearcache`

**Comment by archfz** [comment]
Still doesn't work. I feel like it's somehow related to the name containing / or -

**Comment by kylekatarnls** [comment]
I use this for years in many project in composer 1 and 2, I ran this exact command successfully before posting it. You can also try to put `"pdepend/pdepend": "dev-fix/trailing-comma as 2.11.0"` manually in the "require" or "require-dev" of your composer.json, it does the same and the ` as 2.11.0` should help if using via PHPMD for instance which requires pdepend to be 2.x.

**Comment by archfz** [comment]
Confirmed working.
